docs/browser.md

```diff
@@ -0,0 +1,14 @@
+## jquery.flot.browser.js
+
+This plugin is used to make available some browser-related utility functions.
+
+### getPageXY
+Use getPageXY to obtain the viewable area of the page as set by the scroll bars.
```

Comment:
Calculates the pageX and pageY using the screenX, screenY properties of the event and the scrolling of the page. This is needed because the pageX and pageY properties of the event are not correct while running tests in Edge.

Reply:
Done